src/hera/shared/_base_model.py

```diff
@@ -10,16 +13,19 @@ class Config:
         """
 
         allow_population_by_field_name = True
-        """support populating Hera object fields via keyed dictionaries"""
+        """support populating Hera object fields via keyed dictionaries."""
 
         allow_mutation = True
-        """supports mutating Hera objects post instantiation"""
+        """supports mutating Hera objects post instantiation."""
 
         use_enum_values = True
-        """supports using enums, which are then unpacked to obtain the actual `.value`, on Hera objects"""
+        """supports using enums, which are then unpacked to obtain the actual `.value`, on Hera objects."""
 
         arbitrary_types_allowed = True
-        """supports specifying arbitrary types for any field to support Hera object fields processing"""
+        """supports specifying arbitrary types for any field to support Hera object fields processing."""
 
         smart_union = True
-        """uses smart union for matching a field's specified value to the underlying type that's part of a union"""
+        """uses smart union for matching a field's specified value to the underlying type that's part of a union."""
+
+        extra = Extra.forbid
```

Comment:
Maybe we can turn it off when interacting with the server to solve the underlying issue?

Reply:
Yea, probably.. I am surprised that the fields on the return payload _may_ not match the fields on the autogen code though :(